src/kiwixapp.cpp

```diff
@@ -196,6 +196,7 @@ void KiwixApp::setSideBar(KiwixApp::SideBarType type)
     switch(type) {
         case SEARCH_BAR:
             mp_mainWindow->findChild<TocSideBar*>("tocsidebar")->getFindLineEdit()->setFocus();
+            break;
```

Comment:
Because the find in page search bar will no longer be in the sidebar (#327), I removed these lines in the PR https://github.com/kiwix/kiwix-desktop/pull/347/commits/31c0b53418f712de4500ab681809ea9f76ebd817, so I don't think that it's important to add this ```break```.

Reply:
I've removed the commit